go.mod

```diff
@@ -252,13 +252,15 @@ replace (
 	k8s.io/cloud-provider => ./staging/src/k8s.io/cloud-provider
 	k8s.io/cluster-bootstrap => ./staging/src/k8s.io/cluster-bootstrap
 	k8s.io/code-generator => ./staging/src/k8s.io/code-generator
+	k8s.io/code-generator/examples => ./staging/src/k8s.io/code-generator/examples
```

Comment:
was this required? I really don't want _any_ trace of the test-only / examples / mock modules from the root, or I'm afraid their deps will start to get pulled into the root module's graph